src/modules/admissions/views/EditInterview.tsx

```diff
@@ -68,6 +71,25 @@ export const EditInterview: React.FC = () => {
     },
   })
 
+  const saveKeyHandler = useCallback(
+    (e: KeyboardEvent) => {
+      if (e.key === 's' && (e.ctrlKey || e.metaKey)) {
+        e.preventDefault()
+        handleSaveNotes()
+      }
+    },
+    [notesEditor, discussionEditor, data]
+  )
```

Comment:
maybe we could make this more verbose? I get what it's doing but it's hard to read what it's used for just by looking at the handler. Maybe I'm just being a noob.

Reply:
Not sure how we can make it more verbose? An event handler that triggers a `handleSaveNotes` if the correct keyboard combination is hit